ktlint-ruleset-standard/src/test/kotlin/com/pinterest/ktlint/ruleset/standard/TrailingCommaRuleTest.kt

```diff
@@ -1,119 +1,21 @@
 package com.pinterest.ktlint.ruleset.standard
 
-import com.pinterest.ktlint.ruleset.experimental.trailingcomma.TrailingCommaRule
-import com.pinterest.ktlint.ruleset.experimental.trailingcomma.TrailingCommaRule.Companion.allowTrailingCommaOnCallSiteProperty
-import com.pinterest.ktlint.ruleset.experimental.trailingcomma.TrailingCommaRule.Companion.allowTrailingCommaProperty
 import com.pinterest.ktlint.test.KtLintAssertThat.Companion.assertThat
 import com.pinterest.ktlint.test.LintViolation
 import org.junit.jupiter.api.Test
 
 class TrailingCommaRuleTest {
```

Comment:
Remaining in this class are the tests combining both rules.

Reply:
The rules do not affect each other. So it does not make sense to keep them as combined test. I would say the test needs to be stripped down and splitted into multiple tests, each of which only tests one rule.